### ISSUE TITLE
Sign in not possible

### ISSUE THREAD

**Issue by Geo-Joe** [cgeo/cgeo#7323]
##### Detailed steps causing the problem:
- I open Cgeo and can not sign in.
- this means I cant update caches or update live-map to show new caches !
- Correct Account data is used.
- i am opening Settings and try to sign in manually not possible. in addition I checked sign in data, it is correct. Authorisim was not possible, I get the massage: sign in failure, connection to geocaching.com is not possible (server down or connection inactive)
- Network is still runing, I try it with WLan and mobile data.
- Browser sign in at Geocaching.com page is possible with both connection types.

- I updated today c:geo but this failure still exist.

##### Actual behavior after performing these steps:

This failure exist since approximate 2-3weeks. Before I open c:geo and it connected automatically to Geocaching.com.
So cache update and live-map update was possible without any problems.

##### Version of c:geo used:

2019.02.23

##### Is the problem reproducible for you?
Yes/No

Yes (it is permanent)

##### System information:
```
--- System information ---
Device: GT-I9305 (m3xx, samsung)
Android version: 4.3
Android build: JSS15J.I9305XXUENH1
c:geo version: 2019.02.23
Google Play services: enabled - 15.0.90 (000306-231259764)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide own/found: false
Map strategy: auto
HW acceleration: enabled (default state)
System language: de_DE
System date format: dd.MM.yyyy
Debug mode active: no
System internal c:geo dir: /data/data/cgeo.geocaching (916,0 MB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (896,0 MB free) external
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (896,0 MB free) external
Database: /data/data/cgeo.geocaching/databases/data (117,9 MB) on system internal storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
   geocaching.com: Not logged in (Anmeldung…)
Geocaching.com date format: MM/dd/yyyy
Installed c:geo plugins:  none
--- End of system information ---
```

##### Other comments and remarks:
so long as sign in is not possible c:geo is quite useless for me.

**Comment by SammysHP** [comment]
What happens when you open https://www.geocaching.com in a web browser on your phone? If you have more than one, please try all installed browsers.

**Comment by Geo-Joe** [comment]
I use normally (95 percent) firefox and the rest Chrome.
Firefox: webpage is normally opening
Chrome: failure Code, this is not a secured connection

SammysHP <notifications@github.com> schrieb am So., 24.02.2019, 14:34:
>
> What happens when you open https://www.geocaching.com in a web browser on
your phone? If you have more than one, please try all installed browsers.

**Comment by SammysHP** [comment]
> Chrome: failure Code, this is not a secured connection

This is the problem. Your phone does not support the encryption method that is required to use geocaching.com. We have a workaround for versions 4.1 – 4.3 that should work in most situations. As the problem occurred for you also with the previous release of c:geo I think that it is not related to the recent target API change of the new release.

For more information read http://www.cgeo.org/faq#7078

**Comment by SammysHP** [comment]
I've just seen that geocaching.com updated their certificate on January 25. Does this coincide with your problems? They might have disabled additional ciphers required for the legacy workaround.

**Comment by bekuno** [comment]
I have a simular device and could log in:

--- System information ---
Device: GT-I9300 (m0xx, samsung)
Android version: 4.3
Android build: JSS15J.I9300XXUGOL2
c:geo version: 2019.02.23
Google Play services: enabled - 15.0.90 (000306-231259764)
Low power mode: inactive
Compass capabilities: yes
Rotation vector sensor: present
Orientation sensor: present
Magnetometer & Accelerometer sensor: present
Direction sensor used: rotation vector
Hide own/found: false
Map strategy: auto
HW acceleration: enabled (default state)
System language: de_DE
System date format: dd.MM.yyyy
Debug mode active: no
System internal c:geo dir: /data/data/cgeo.geocaching (8,1 GB free) internal
User storage c:geo dir: /storage/emulated/0/cgeo (8,0 GB free) external
Geocache data: /storage/emulated/0/Android/data/cgeo.geocaching/files/GeocacheData (8,0 GB free) external
Database: /data/data/cgeo.geocaching/databases/data (264,0 KB) on system internal storage
Fine location permission: granted
Write external storage permission: granted
Geocaching sites enabled:
geocaching.com: Logged in (Anmeldung OK) / PREMIUM
opencaching.de: Not logged in (Nicht autorisiert)
Geocaching.com date format: dd.MM.yy
Installed c:geo plugins: none
--- End of system information ---

**Comment by bekuno** [comment]
After succesfull connection the message says "success ...", but in the settings window the text was not changed ("not yet connected"). After reloading the settings site it shows as activated.

**Comment by Lineflyer** [comment]
> After succesfull connection the message says "success ...", but in the settings window the text was not changed ("not yet connected"). After reloading the settings site it shows as activated.

This is known and should be listed in the issues here somewhere.

For the general problem described by the user I agree, that this might be caused by Android 4.3 and its non supported certificates / encryption methods.

@SammysHP I sadly don't have a phone with < Android 4.4 

@Geo-Joe Can you please try the longpress reset as described here: http://faq.cgeo.org/#login-failed

**Comment by bekuno** [comment]
@Lineflyer See my system imformation: Android version: 4.3

**Comment by Geo-Joe** [comment]
I tried the longpress reset as described. Result: same failure still
existing (no sign in possible).
After that, I reset C:Geo to factury setting, but after all it still not
work.

I am wondering why I can log in at the homepage of geocaching.com with
firefox but C:Geo can´t sign in.
In addition it is strange that Bernd also use Android 4.3 and had no
problems with the sign in process.

Am So., 24. Feb. 2019 um 21:36 Uhr schrieb Lars <notifications@github.com>:

**Comment by Lineflyer** [comment]
> I am wondering why I can log in at the homepage of geocaching.com with
> firefox but C:Geo can´t sign in.
> In addition it is strange that Bernd also use Android 4.3 and had no
> problems with the sign in process.

It is important whether your device supports the needed encryptions. That should not the case on most devices < Android 4.4

The other point is the workaround we implemented for <4.4. Does someone know the technical details of this?

**Comment by Lineflyer** [comment]
Closing as there was no update on this.
The current status is, that c:geo supports Android 4.1+ whereas logins might fail on Android <4.4 if our workaround does not work on some devices. Nothing we can do about this IMHO.
